frontend/src/types/media.ts

```diff
@@ -46,6 +46,8 @@ export interface Media {
   fields_matched?: string[]
 
   mature: boolean
+  sensitivity: string[]
```

Comment:
Nit: Should we type this more explicitly with the possible values? Something like:

```ts
type SensitiveType = 'user_reported_sensitive' | 'provider_supplied_sensitive' | 'sensitive_text';
type SensitivityList = SensitiveType[];

```